docs/_peoples/jinglei-ren.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Jinglei REN (任晶磊)
 site: https://people.persper.org/jinglei/
-bio: It is not truly great work if everyone thinks it is.
+bio: 清华大学计算机系博士，前微软亚洲研究院研究员，曾在斯坦福大学、卡内基梅隆大学担任访问学者。在软件系统、软件工程领域从事多年前沿研究，多篇论文发表在FSE、OSDI等顶尖国际学术会议上；积极贡献开源社区，代码被多个用户广泛的开源项目所使用；亦参与过微软重大工程项目的设计与实施。后创建Persper基金会，与FreeBSD早期领导者Poul-Henning Kamp、MIT斯隆管理学院教授Eric von Hippel、加州大学伯克利分校软件工程领域知名教授Armando Fox等一道，致力于将“知识股权”引入开源项目。现为硅谷科技公司Merico创始人和CEO，专注于研发程序代码库分析技术，测量程序员对软件开发活动的贡献，服务于全球企业的软件工程管理和人才管理。
```

Comment:
A bio should be kept short, and the shorter the better.

This bio is too long, could you please shorten it for around 30 - 70 words?